lib/pharos/scripts/configure-essentials.sh

```diff
@@ -2,13 +2,26 @@
 
 set -e
 
+. /usr/local/lib/pharos.sh
```

Comment:
My `/usr/local/lib` just has a bunch of `.so` and `.a` files and subdirs, maybe `/usr/local/lib/pharos/util.sh`?